clients/fides-js/docs/interfaces/Fides.md

```diff
@@ -75,34 +75,25 @@ A `Fides.consent` value showing the user has opted-in to analytics, but not mark
 
 > `optional` **fides\_string**: `string`
 
-User's current consent string(s) combined into a single value. The string
-consists of three parts separated by commas in the format:
-`TC_STRING,AC_STRING,GPP_STRING` where:
+User's current consent string(s) combined into a single value. This is used by
+FidesJS to store IAB consent strings from various frameworks such as TCF, GPP,
+and Google's "Additional Consent" string. Additionally, we support passing a
+Notice Consent string, which is a base64 encoded string of the user's Notice
+Consent preferences. See [FidesOptions.fides_string](FidesOptions.md#fides_string) for more details.
+
+The string consists of four parts separated by commas in the format:
+`TC_STRING,AC_STRING,GPP_STRING,NC_STRING` where:
 
 - TC_STRING: IAB TCF (Transparency & Consent Framework) string
-- AC_STRING: Google's Additional Consent string, derived from TC_STRING
+- AC_STRING: Google's Additional Consent string
 - GPP_STRING: IAB GPP (Global Privacy Platform) string
-
-Note: The AC_STRING can only exist if TC_STRING exists, as it's derived from it.
-When GPP is enabled, the GPP_STRING portion is automatically initialized during
-FidesJS initialization, either preserving any existing GPP string or using a
-default value. The GPP_STRING is independent and can exist with or without the
-other strings.
```

Comment:
moved a lot of the docs for setting the string to the FidesOptions docs since that's more appropriate. These docs here are mostly for reading the string.